### ISSUE TITLE
Please finalize the realistic human models so that they are at least in an usable state.

### ISSUE THREAD

**Issue by eEight-dev** [sboxgame/issues#3956]
### For?

S&Box

### What can't you do?

Myself and other members of the community would very much like to use the realistic human models for various needs and projects. The citizen model isn't a "one size fits all" kind of solution for all the games that are out there. The proportions are very different from the realistic humans, which makes a very big difference in FPS games where it shouldn't be that easy to get a headshot on someone. It's also a stylized character and it doesn't often blend well with more realistic style environments, models, etc. I could come up with more examples...

### How would you like it to work?

If I remember correctly, Max mentioned in the discord that the weights and bones need to be adjusted. That would be awesome if done sometime soon. I also believe that if models were to be ready, the community could come up with some open-source animations that everyone could use. It would also be great if the source files and relevant .vmdls of the models were to be made public so that people don't have to rely on exporting from modeldoc to get the files.

### What have you tried?

I have asked when we may see them fixed quite some time ago, but I was told they are very low priority.

### Additional context

I think they were released in very early 2023, it has been many months since then and they are still not in an usable state, meanwhile we kept getting more stuff for citizen, completely ignoring the existence of the realistic humans.

**Comment by dinga989** [comment]
I must also express the need for these to be fixed. I don't want to make clothing for a broken citizen model, then it gets fixed, then my clothing becomes incompatible for example.

**Comment by e1m1j** [comment]
I agree on the absurdity that these have just been left to gather dust but they're probably not a priority right now. You're almost better off making and rigging your own characters at this point because of the uncertainty of the realistic models.

**Comment by MaxLebled** [comment]
We will be working more on them in the future, but we have other art and pipeline related problems that need to be tackled now before we can resume work on the Humans. Not resolving them beforehand would mean, among other things, having to solve some issues _twice_, which would drastically slow down our work on both fronts.

Don't worry — they're not getting abandoned. They're just not a priority right now, and they structurally cannot be... yet!
